libs/go/zmssvctoken/token.go

```diff
@@ -229,8 +237,9 @@ type tokenBuilder struct {
 
 // NewTokenBuilder returns a TokenBuilder implementation for the specified
 // domain/name, with a private key (PEM format) and its key-version. The key-version
-// should be the same string that was used to register the key with Athenz.
-func NewTokenBuilder(domain, name string, privateKeyPEM []byte, keyVersion string) (TokenBuilder, error) {
+// should be the same string that was used to register the key with Athenz. The key-service
+// (optional) should be the service name where the public key is stored.
+func NewTokenBuilder(domain, name string, privateKeyPEM []byte, keyVersion, keyService string) (TokenBuilder, error) {
```

Comment:
Why was keyService added to the NewTokenBuilder api? This has 2 issues that we should avoid:
a) breaks existing clients who already use the api. From client's perspective, they're always going to pass "" since no client has access to the zms/zts keys to sign anything
b) it's an optional component so it must be treated like all optional components in the ntoken with the appropriate SetKeyService endpoint. (similar to SetHostname/SetIPAddress,etc)

Reply:
@havetisyan That's a good point. I have updated the code so that this parameter is considered optional and have removed it from the NewTokenBuilder api. Can you please take a look again? Thanks.